client/src/main/kotlin/io/spine/examples/pingh/client/DesktopClient.kt

```diff
@@ -26,73 +26,31 @@
 
 package io.spine.examples.pingh.client
 
-import com.google.protobuf.Duration
 import com.google.protobuf.Message
 import io.grpc.ManagedChannelBuilder
 import io.spine.base.CommandMessage
 import io.spine.base.EntityState
 import io.spine.base.EventMessage
 import io.spine.base.EventMessageField
 import io.spine.base.Field
-import io.spine.base.Time.currentTime
 import io.spine.client.Client
 import io.spine.client.ClientRequest
-import io.spine.client.ConnectionConstants.DEFAULT_CLIENT_SERVICE_PORT
 import io.spine.client.EventFilter.eq
 import io.spine.client.Subscription
 import io.spine.core.UserId
-import io.spine.examples.pingh.github.Username
-import io.spine.examples.pingh.mentions.GitHubClientId
-import io.spine.examples.pingh.mentions.MentionId
-import io.spine.examples.pingh.mentions.MentionView
-import io.spine.examples.pingh.mentions.UserMentions
-import io.spine.examples.pingh.mentions.UserMentionsId
-import io.spine.examples.pingh.mentions.buildBy
-import io.spine.examples.pingh.mentions.command.MarkMentionAsRead
-import io.spine.examples.pingh.mentions.command.SnoozeMention
-import io.spine.examples.pingh.mentions.command.UpdateMentionsFromGitHub
-import io.spine.examples.pingh.mentions.event.MentionRead
-import io.spine.examples.pingh.mentions.event.MentionSnoozed
-import io.spine.examples.pingh.mentions.event.MentionsUpdateFromGitHubCompleted
-import io.spine.examples.pingh.mentions.event.RequestMentionsFromGitHubFailed
-import io.spine.examples.pingh.sessions.SessionId
-import io.spine.examples.pingh.sessions.buildBy
-import io.spine.examples.pingh.sessions.command.LogUserIn
-import io.spine.examples.pingh.sessions.command.LogUserOut
-import io.spine.examples.pingh.sessions.event.UserLoggedIn
-import io.spine.examples.pingh.sessions.event.UserLoggedOut
-import io.spine.protobuf.Durations2
-import java.util.UUID
 import kotlin.reflect.KClass
 
 /**
  * Interacts with [Pingh server][io.spine.examples.pingh.server] via gRPC.
  *
- * By default, client will open channel to 'localhost:[50051]
- * [io.spine.client.ConnectionConstants.DEFAULT_CLIENT_SERVICE_PORT]'.
+ * @param userId user on whose behalf client requests are made.
  */
-@Suppress("TooManyFunctions") // The client must contain multiple methods
-// to interact with the server, which does not enable Detekt.
-public class DesktopClient(
-    address: String = "localhost",
-    port: Int = DEFAULT_CLIENT_SERVICE_PORT
+internal class DesktopClient(
+    address: String,
```

Comment:
There are two more parameters to document.

Please review this changeset for the similar issues.